src/orca-loader/Elements.res

```diff
@@ -239,7 +242,10 @@ let make = (
           ->JSON.Encode.object
         let message =
           [
-            ("paymentElementCreate", (componentType == "payment")->JSON.Encode.bool),
+            (
+              "paymentElementCreate",
+              componentType->Utils.isComponentTypeForPaymentElementCreate->JSON.Encode.bool,
```

Comment:
can you change this name to `isPaymentElement`
```suggestion
              componentType->Utils.isPaymentElement->JSON.Encode.bool,
```

Reply:
I don't think `isPaymentElement` will be appropriate because the function will return true not only for payment element but for the widgets as well (applePay, googlePay and payPal)